packages/app/webpack.config.js

```diff
@@ -23,13 +29,23 @@ module.exports = {
         ]
     },
     resolve: {
-        extensions: ['.ts', '.tsx', '.mjs', '.js', '.json']
+        extensions: ['.ts', '.tsx', '.mjs', '.js', '.jsx', '.json'],
+        alias:{
+            "react-dom":"@hot-loader/react-dom"
+        }
     },
     plugins: [
+        //new webpack.DefinePlugin({'process.env.NODE_ENV': JSON.stringify(process.env.NODE_ENV)}),
+        new webpack.HotModuleReplacementPlugin(),
+        new webpack.NamedModulesPlugin(),
```

Comment:
not needed in webpack@4+ AFAIK:
https://stackoverflow.com/questions/40861768/webpack-plugin-for-module-friendly-names-path-instead-of-a-number